.github/workflows/release.yaml

```diff
@@ -93,7 +99,7 @@ jobs:
           ${{ env.AWS_DOCKER_TAG }}
           ${{ env.AWS_LATEST_TAG }}
         build-args: |
-          BASE_IMAGE=mosaicml/pytorch:2.4.0_cu124-python3.11-ubuntu20.04-aws
+          BASE_IMAGE=mosaicml/llm-foundry:2.4.0_cu124-latest
```

Comment:
how is base image the same image ur building...?

Reply:
They're different images, the images built here are 
`mosaicml/llm-foundry:release_${TAG_NAME}"`
to keep up with foundry releases

Reply:
wait yeah, how are we building off the foundry images? that should be the result of this right

Reply:
ohh. Huh, I guess the behavior is slightly weird.

so https://github.com/mosaicml/llm-foundry/blob/main/.github/workflows/docker.yaml is what builds `mosaicml/llm-foundry:2.4.0_cu124-latest` (which is largely cached), and then `https://github.com/mosaicml/llm-foundry/blob/main/.github/workflows/release.yaml` builds from that image except with the foundry directory in it...

but like isn't that fine?

Reply:
But currently the release workflow builds on the pytorch image, not the foundry image...i guess the foundry image does exist though...

Reply:
I don't think its worth optimizing release workflow to add this complication. This feels a bit weird to me to build on top of the foundry build image...

Reply:
^ agree

Reply:
This can have some unintended complications -- for example when doing foundry release, suppose I create the release branch but don't publish the release and kick off the release workflow yet. Then, in between, someone else merges some breaking PR. Then I launch this new release workflow which builds on the cached latest foundry image that contains the bad commit....right?

Reply:
Ok, sounds fair, will close pr